test/testdata/configs/misspell_custom.yml

```diff
@@ -0,0 +1,7 @@
+linters-settings:
+  misspell:
+    extra-words:
+      - typo: "iff"
```

Comment:
I'm not sure if `iff` is a good example of a typo. "iff" can stand for ["If and only if"](https://en.wikipedia.org/wiki/If_and_only_if).

Some time ago, I created a [PR](https://github.com/dominikh/go-tools/pull/1375) to fix `iff` to `if`, but it was rejected.

Reply:
https://go.dev/wiki/Spelling